### ISSUE TITLE
Compliance to RFC 4960

### ISSUE THREAD

**Issue by soniabha-intc** [ishidawataru/sctp#45]
Hi , is this library compatible with the linux SCTP kernel module which supports 4960. We plan to use this with Centos7.6 or Ubuntu 20.04

**Comment by shanehooker** [comment]
yes. There are limitations, from the APIs, as to what SCTP paramaters and options can be set using this package but there are other ways around that.

**Comment by soniabha-intc** [comment]
Are such limitations in this library documented anywhere?
